config/app.php

```diff
@@ -65,8 +65,6 @@
 
 	'url' => env('APP_URL', 'http://localhost'),
 
-	'asset_url' => env('ASSET_URL'),
-
```

Comment:
Are we sure that we do not need this config option? We use the method `asset()` in some Blade templates below `view/`.

Even if we do not use it, I assume that is is better to set this variable to `null`. I cannot assess what happens if the variable is unset. It is used here https://github.com/laravel/framework/blob/cd6e76fe864161af7307be7abb6d4d18cbe064ca/src/Illuminate/Routing/RoutingServiceProvider.php#L62-L66 I guess this will throw an exception, if the array key is not set.